src/main/kotlin/rx/lang/kotlin/namespace.kt

```diff
@@ -1,70 +0,0 @@
-/**
- * Copyright 2013 Netflix, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package rx.lang.kotlin
-
-import rx.Observable
-import rx.Observable.OnSubscribe
-import rx.Subscriber
-
-
-public fun<T> Function1<Subscriber<in T>, Unit>.asObservable(): Observable<T> {
```

Comment:
Your work is amazing, but there are people that are already using this.

So instead of deleting those extensions we could marked as deprecated and then we'll delete it in the next iteration.

That also will help people to understand the new API

Could I raise a PR against your branch with the afore mentioned deprecations?